### PR TITLE
dodalam zmiany w tripDetailView

### PR DESCRIPTION
**Pull request by albinagekova**

### CODE REVIEW

File: vue-front/src/components/AddTripForm.vue [hunk 1]
```diff
@@ -24,7 +24,7 @@
     <div class="buttonContainer">
       <custom-button
         label="Dodaj"
-        @click="() => addTrip($route.params)"
+        @click="addTrip"
```
Comment (GabiBack) [inline vue-front/src/components/AddTripForm.vue]: @click="() => addTrip($route.params)"

File: vue-front/src/components/AddTripForm.vue [hunk 2]
```diff
@@ -48,9 +48,8 @@ export default {
     CustomDatePicker
 },
   methods: {
-    addTrip: async function(routeParams) {
-      const {userId} = routeParams
-      const trip = await fetch(`http://localhost:8080/trip/${userId}`, {
+    addTrip: async function() {
+      const trip = await fetch('http://localhost:8080/trip/', {
```
Comment (GabiBack) [inline vue-front/src/components/AddTripForm.vue]: const trip = await fetch(`http://localhost:8080/trip/${userId}`, {

File: vue-front/src/components/AddTripForm.vue [hunk 3]
```diff
@@ -48,9 +48,8 @@ export default {
     CustomDatePicker
 },
   methods: {
-    addTrip: async function(routeParams) {
-      const {userId} = routeParams
-      const trip = await fetch(`http://localhost:8080/trip/${userId}`, {
+    addTrip: async function() {
```
Comment (GabiBack) [inline vue-front/src/components/AddTripForm.vue]: pod tą linijką dodaj jeszcze to: 
const {userId} = routeParams

File: vue-front/src/router/index.js [hunk 4]
```diff
@@ -34,12 +34,12 @@ const routes = [
     component: WelcomeView,
   },
   {
-    path: '/trips/:userId',
+    path: '/trips',
```
Comment (GabiBack) [inline vue-front/src/router/index.js]: path: '/trips/:userId',

File: vue-front/src/router/index.js [hunk 5]
```diff
@@ -34,12 +34,12 @@ const routes = [
...
     name: 'trips',
     component: TripsView,
   },
   {
-    path: '/trip/:id',
+    path: '/trips/:id',
```
Comment (GabiBack) [inline vue-front/src/router/index.js]: path: '/trip/:id',

File: vue-front/src/views/TripsView.vue [hunk 6]
```diff
@@ -16,7 +16,7 @@
           <th><span>Dzień rozpoczęcia</span></th>
           <th><span>Dzień zakończenia</span></th>
         </tr>
-        <tr v-for="({ place, id, dateStart, dateEnd }) in trips" v-bind:key="id" @click="$router.push(`/trip/${id}`)">
+        <tr v-for="({ place, id, dateStart, dateEnd }) in trips" v-bind:key="id" @click="$router.push(`/trips/${id}`)">
```
Comment (GabiBack) [inline vue-front/src/views/TripsView.vue]: zamiast: @click="$router.push(`/trips/${id}
ma być: @click="$router.push(`/trip/${id}

File: vue-front/src/views/TripsView.vue [hunk 7]
```diff
@@ -60,9 +60,8 @@ function readFileAsync(file) {
 }
 
 const modalOpen = ref(false)
-
-async function fetchTrips(userId) {
-  const response = await fetch(`http://localhost:8080/user/trip/${userId}`)
+async function fetchTrips() {
```
Comment (GabiBack) [inline vue-front/src/views/TripsView.vue:63]: async function fetchTrips(userId) {
   const response = await fetch(`http://localhost:8080/user/trip/${userId}`)

File: vue-front/src/views/TripsView.vue [hunk 8]
```diff
@@ -107,8 +106,7 @@ export default {
     }
   },
   async mounted() {
-    const {userId} = this.$route.params
-    await fetchTrips(userId)
+    fetchTrips()
```
Comment (GabiBack) [inline vue-front/src/views/TripsView.vue]: const {userId} = this.$route.params
     await fetchTrips(userId)

File: vue-front/src/views/WelcomeView.vue [hunk 9]
```diff
@@ -30,8 +30,7 @@ export default {
 },
   methods: {
     handleButtonClick: function() {
-      const {userId} = this.$route.params
-      this.$router.push(`/trips/${userId}`)
+      this.$router.push(`/trips`)
```
Comment (GabiBack) [inline vue-front/src/views/WelcomeView.vue:33]: const {userId} = this.$route.params
       this.$router.push(`/trips/${userId}`)

**Comment by albinagekova** [comment]
Poprawilam w tych dwoch commitach

**Review by GabiBack — APPROVED** [review]

**Pull request merged**